boards/shields/adafruit_winc1500/adafruit_winc1500.overlay

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2020 Kim Bøndergaard, <kim.bondergaard@prevas.dk>, Prevas A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+	cs-gpios = <&arduino_header 16 0>;			/* D10 */
+
+        winc1500@0 {
```

Comment:
nit: the indent doesn't seem right here - everything should be indented with a one or two tabs.

Reply:
Indenting fixed